.github/workflows/benchmark-runner.yml

```
@@ -143,7 +150,7 @@ jobs:
       benchmark_runner_group_total: ${{ strategy.job-total }}
 
   benchmark-search-oss-standalone-profiler:
-    if: inputs.profiler
+    if: ${{ inputs.extended && (inputs.allowed_setup == '' || inputs.allowed_setup == 'standalone') }}
```

Shouldn't use `inputs.profiler` ?
```suggestion
    if: ${{ inputs.profiler && (inputs.allowed_setup == '' || inputs.allowed_setup == 'standalone') }}
```